src/skibidi_orm/query_engine/model/base.py

```diff
@@ -0,0 +1,117 @@
+""" A model class for mapping database table """
+
+from pydantic import BaseModel
+from pydantic.main import ModelMetaclass
+from typing import Any, Tuple, Dict, List
```

Comment:
you can write typehints without importing from typing Tuple, Dict, List. You can write dict[str, str] and it works. You don't have to change it, this is ok too.

Reply:
Ok, I will change